examples/tv-casting-app/tv-casting-common/core/CastingPlayer.h

```diff
@@ -197,6 +262,17 @@ class CastingPlayer : public std::enable_shared_from_this<CastingPlayer>
     std::vector<memory::Strong<Endpoint>> mEndpoints;
     ConnectionState mConnectionState = CASTING_PLAYER_NOT_CONNECTED;
     CastingPlayerAttributes mAttributes;
+    IdentificationDeclarationOptions mIdOptions;
+    /**
+     * Flag to indicate whether or not the Commissionee (tv-casting-app) is enabled for the Commissioner-Generated Passcode flow.
+     */
+    bool mUdcCommissionerPasscodeEnabled = false;
+    /**
+     * Flag to indicate whether or not the Commissionee has obtained the Commissioner Passcode from the user and is therefore ready
+     * for commissioning.
+     */
+    bool mUdcCommissionerPasscodeReady = false;
+    CommissionerDeclarationHandler * mCommissionerDeclarationHandler;
```

Comment:
Since all of these attributes are about a UDC request being sent to the CastingPlayer, it would be cleaner to encapsulate all of these in a new class, say `CastingPlayerUDCContext` in a new header. Then, just refer to that from here, if you need to.

Reply:
I don't see any changes corresponding to this one. Can you take a look here?